sources/packages/api/src-sql/Student/Add-sin-consent-col.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS sin_consent BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Default is for the purpose of handling existing data, we must drop default after adding the column.

Reply:
I have dropped the default.